market/fakelm/lmimpl.go

```diff
@@ -76,15 +76,19 @@ func (l *LMRPCProvider) SectorsStatus(ctx context.Context, sid abi.SectorNumber,
 	// TODO: Add snap, Add open_sector_pieces
 
 	var ssip []struct {
-		PieceCID *string `db:"piece_cid"`
-		DealID   *int64  `db:"f05_deal_id"`
-		DDOPAM   *string `db:"ddo_pam"`
-		Complete bool    `db:"after_commit_msg_success"`
-		Failed   bool    `db:"failed"`
-		SDR      bool    `db:"after_sdr"`
-		PoRep    bool    `db:"after_porep"`
-		Tree     bool    `db:"after_tree_r"`
-		IsSnap   bool    `db:"is_snap"`
+		PieceCID        *string `db:"piece_cid"`
+		DealID          *int64  `db:"f05_deal_id"`
+		DDOPAM          *string `db:"ddo_pam"`
+		Complete        bool    `db:"after_commit_msg_success"`
+		Failed          bool    `db:"failed"`
+		SDR             bool    `db:"after_sdr"`
+		PoRep           bool    `db:"after_porep"`
+		Tree            bool    `db:"after_tree_r"`
+		IsSnap          bool    `db:"is_snap"`
+		Encode          bool    `db:"after_encode"`
+		SnapProve       bool    `db:"after_prove"`
+		SnapCommit      bool    `db:"after_prove_msg_success"`
+		SnapMoveStorage bool    `db:"after_move_storage"`
```

Comment:
Don't those appear as null when the sector is not in the snap pipeline?

Reply:
Thus COALESCE with FALSE so it can either be true or false.